image/config.go

```diff
@@ -31,6 +31,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+// SpecURL is image-specs
+var SpecURL = "https://github.com/opencontainers/image-spec"
```

Comment:
`config.go` is about manipulating the image-config object, so I don't think these URLs are a good fit.  I expect we'll want a new file for them, and had [floated `project.go` earlier][1].

[1]: https://github.com/opencontainers/image-tools/pull/88#issuecomment-262326315